docs/source/topics/frontera-settings.rst

```diff
@@ -437,6 +437,96 @@ Default: ``timedelta(days=1)``
 Time between document visits, expressed in ``datetime.timedelta`` objects. Changing of this setting will only affect
 documents scheduled after the change. All previously queued documents will be crawled with old periodicity.
 
+.. _cassandra-settings:
+
+Cassandra
+---------
+
+
+.. setting:: CASSANDRABACKEND_DROP_ALL_TABLES
+
+CASSANDRABACKEND_DROP_ALL_TABLES
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``False``
+
+Set to ``True`` if you need to drop of all DB tables on backend instantiation (e.g. every Scrapy spider run).
+
+.. setting:: SQLALCHEMYBACKEND_ENGINE
+
+CASSANDRABACKEND_CLUSTER_IPS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default:: ``['127.0.0.1']``
+
+Set IPs from Cassandra Cluster. Default is localhost. To assign more than one IP use this Syntax: ``['192.168.0.1', '192.168.0.2']``
+
+CASSANDRABACKEND_CLUSTER_PORT
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default:: ``9042``
+
+Set port from Cassandra Cluster / Nodes
+
+
+CASSANDRABACKEND_GENERATE_STATS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default:: ``False``
+
+Set this to true if you want to create an extra Table for stats collection. In this table there will be pages crawled, links queued etv. counted up.
```

Comment:
It's not only about creation, but also updating stats there. So, I would generalize it as `to use and extra table for statistics collection`.
There is a misspell in `etv.`